app/Http/Controllers/QuizController.php

```diff
@@ -85,4 +85,15 @@ public function createSubmission(Request $request, Quiz $quiz): RedirectResponse
 
         return redirect("/submissions/{$submission->id}/");
     }
+
+    public function assign(Request $request, Quiz $quiz)
+    {
+        $user = $request->user();
+        $quiz->assignedUsers()->attach($user);
+        $quiz->save();
+
+        return redirect()
+            ->back()
+            ->with("status", "Zostałeś przypisany do testu");
```

Comment:
```suggestion
            ->with("status", "Zostałeś przypisany do testu");
```

How about girls? "Zostałeś/aś"? Ormaybe something different?

Reply:
Done